NEWS.md

```diff
@@ -20,7 +20,7 @@ This release is in development. For a stable release install 1.3.5 from CRAN.
 * [pak](https://pak.r-lib.org/) is now suggested for installing the developmental version of the package. By @jamesmbaazam in #407 and reviewed by @seabbs. This has been successfully tested on MacOS Ventura, Ubuntu 20.04, and Windows 10. Users are advised to use `remotes::install_github("epiforecasts/EpiNow2")` if `pak` fails and if both fail, raise an issue.
 * `dist_fit()`'s `samples` argument now takes a default value of 1000 instead of NULL. If a supplied `samples` is less than 1000, it is changed to 1000 and a warning is thrown to indicate the change. By @jamesmbazam in #389 and reviewed by @seabbs.
 * The internal distribution interface has been streamlined to reduce code duplication. By @sbfnk in #363 and reviewed by @seabbs.
-* A small bug has been fixed where the seeding time was too long. By @sbfnk in #413 and reviewed by @seabbs.
+* A small bug has been fixed where the seeding time was one day too long. By @sbfnk in #413 and reviewed by @seabbs.
```

Comment:
Technically that's only true if it's a single delay. If it's e.g. two parametric delays it's two days (because convolution of length X with length Y gives length X+Y-1).

Reply:
ruh roh.

Reply:
Hotfix here: https://github.com/epiforecasts/EpiNow2/commit/3fd7b082185aef9e392be15dcf987e986690d9b9

also 😆 

Also good luck with the course!